src/components/Buttons/Buttons.tsx

```diff
@@ -2,12 +2,28 @@ import React from 'react'
 import styled from 'styled-components'
 import { GaramondWoff } from '../../fonts/Garamond.woff'
 import { GaramondWoff2 } from '../../fonts/Garamond.woff2'
```

Comment:
Maybe it would be better if you make the font names more descriptive depending on their font weights, for example :"GaramondBoldWoff"

Reply:
Agree!